tests/e2e/macvtap_test.go

```diff
@@ -231,6 +234,16 @@ func filterPods(pods []v1.Pod, filterFunction func(v1.Pod) bool) []v1.Pod {
 	return filteredPods
 }
 
+func findMacvtapNamespace() string {
+	// Retrieve manager namespace serching for pods
+	// with label name=kubemacpool and getting the env var POD_NAMESPACE
+	pods, err := clientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{LabelSelector: "name=macvtap-cni"})
+	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should be able to retrieve macvtap pod")
+	ExpectWithOffset(1, pods.Items).To(HaveLen(1), "should have only 1 macvtap pod")
```

Comment:
No expectations in helpers, please.

I would prefer is this function returns `string, error`, and throw an error if there's not a single pod per node.

Plus, the above condition is wrong - e.g. we have 1 pod per node, not a single pod (it's a daemonset).

Reply:
having changed to ds this part is not needed.
Removed. Done